db/migrate/20160619064544_change_memory_reserve_data_type_in_vm.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeMemoryReserveDataTypeInVm < ActiveRecord::Migration[5.0]
+  def up
+    change_column :vms, :memory_reserve, :bigint
```

Comment:
I think its stored in MB?!
At least thats what this PR makes me think: https://github.com/ManageIQ/manageiq/pull/2920

@bdunne do you know more?